src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/CSharp/SymbolExtensions.cs

```diff
@@ -5,21 +5,20 @@
 using System.Collections.Immutable;
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Razor.Language;
 
 namespace Microsoft.CodeAnalysis.Razor;
 
 internal static class SymbolExtensions
 {
-    internal static readonly SymbolDisplayFormat FullNameTypeDisplayFormat =
-        SymbolDisplayFormat.FullyQualifiedFormat
-            .WithGlobalNamespaceStyle(SymbolDisplayGlobalNamespaceStyle.Omitted)
-            .RemoveMiscellaneousOptions(SymbolDisplayMiscellaneousOptions.UseSpecialTypes);
+    internal static string GetDefaultDisplayString(this ISymbol typeSymbol)
```

Comment:
Would be good to add doc comments to each of these with an example of what they produce. GloballyQualified is obvious enough, but I have no idea how "DefaultDisplayString" differs from "FullName".

UPDATE: Okay, looks like "DefaultDisplayString" is just literally the default for whatever Roslyn does in .ToDisplayString(), so examples probably don't make sense, and a doc comment explaining that at least would still be helpful.

Reply:
Added some doccomments that closely resemble the ISymbol.ToDisplayString comments and indicated what it's being called with in these cases